### DISCUSSION TITLE
想请教作者和各位大佬一个关于ex站点的问题

### DISCUSSION THREAD

**Discussion by Tenrxb** [xiaojieonly/Ehviewer_CN_SXJ#1623] [General]
我有里站的权限，用绿E直连eh站点是没有问题的，而且速度还行。但是切换ex站点就不行，挂梯子也不可以，但是在彩E里只能挂梯子进eh和ex，就是很慢，现在也是很纠结，绿E直连表站无压力，彩E能进ex站点但是很慢…

**Comment by ckkkkxkxkxk** [comment]
今天登上去突然发现里站进不去了，然后查看cookie 结果igenous 为mystery  权限也能被收回？

**Reply by wksts** [reply]
我也是

**Reply by R0bbittt** [reply]
igneous过期了，退出登录挂欧美节点重新登进表站再切里站就有新igneous了

**Reply by wksts** [reply]
> igneous过期了，退出登录挂欧美节点重新登进表站再切里站就有新igneous了

解决了，感谢

**Reply by Albert2917** [reply]
> igneous过期了，退出登录挂欧美节点重新登进表站再切里站就有新igneous了
感谢👍
